### PR TITLE
update PicoDet docs

### PR DESCRIPTION
**Pull request by yghstill**

### CODE REVIEW

**Review by qingqing01 — DISMISSED** [review]

**Review by lyuwenyu — DISMISSED** [review]

File: configs/picodet/README.md [hunk 1]
```diff
@@ -5,44 +5,146 @@
 
 We developed a series of lightweight models, which named `PicoDet`. Because of its excellent performance, it is very suitable for deployment on mobile or CPU.
 
-- 🌟 Higher mAP:  The **first** model which within 1M parameter with mAP reaching 30+.
+- 🌟 Higher mAP: the **first** object detectors that surpass mAP **30+** within 1M parameters when the input size is 416.
```
Comment (heavengate) [inline configs/picodet/README.md]: mAP -> mAP(0.5:0.95)

File: configs/picodet/README.md [hunk 2]
```diff
@@ -5,44 +5,146 @@
...
 - 🚀 Faster latency: 114FPS on mobile ARM CPU.
 - 😊 Deploy friendly: support PaddleLite/MNN/NCNN/OpenVINO and provide C++/Python/Android implementation.
 - 😍 Advanced algorithm: use the most advanced algorithms and innovate, such as ESNet, CSP-PAN, SimOTA with VFL, etc.
 
+### Comming soon
+- [ ] More series of model, such as Smaller or larger model.
```
Comment (heavengate) [inline configs/picodet/README.md]: Smaller -> smaller

File: configs/picodet/README.md (hunk 1 above)
Comment (yghstill) [reply configs/picodet/README.md]: done

File: configs/picodet/README.md (hunk 2 above)
Comment (yghstill) [reply configs/picodet/README.md]: done,thx

**Review by heavengate — APPROVED** [review]

**Review by lyuwenyu — APPROVED** [review]

**Pull request merged**